### DISCUSSION TITLE
Note importanti su chiusura anno e passaggio al nuovo

### DISCUSSION THREAD

**Discussion by trinko** [iisgiua/giuaschool#313] [Annunci]
La nuova versione **1.5.0** contiene molte modifiche al codice e l'inserimento di nuovi componenti esterni: ciò ha reso impossibile un test esaustivo di tutte le possibili interdipendenze.

Pertanto si raccomanda di seguire la procedura seguente per evitare problemi:
- utilizzare la versione **1.4.5** ([scarica la 1.4.5](https://github.com/iisgiua/giuaschool/releases/download/v1.4.5/giuaschool-release-v1.4.5.zip)) per il completamento degli scrutini per gli alunni con giudizio sospeso;
- successivamente, sempre utilizzando la versione **1.4.5** effettuare la chiusura d'anno ([vedi maggiori dettagli](https://iisgiua.github.io/giuaschool-docs/task-endYear.html));
- a questo punto aggiornare alla versione **1.5.0**;
- effettuare infine la procedura per il passaggio al nuovo anno scolastico ([vedi maggiori dettagli](https://iisgiua.github.io/giuaschool-docs/task-newYear.html)).

**Comment by sempliciniluigi89** [comment]
Ci potrebbe anticipare anche in elenco tutte le novità che ci saranno nella versione 1.5.0?Siamo curiosi e non vediamo l'ora

**Comment by trinko** [comment]
Le modifiche maggiori riguardano l'infrastruttura dell'applicazione e non sono visibili all'utente: riguardano l'aggiornamento della gestione della sicurezza e l'aggiunta di nuovi componenti che verranno attivati in seguito (riguardanti le email e le notifiche con strumenti vari).
Le uniche modifiche  visibili saranno:
- tolto il limite di 1 carattere alle sezioni, ora sono un testo libero sino a 64 caratteri;
- correzione dei problemi sulle importazioni degli utenti;
- inserimento della procedura di passaggio al nuovo anno.
-
